changelog/@unreleased/pr-4128.v2.yml

```diff
@@ -0,0 +1,7 @@
+type: improvement
+improvement:
+  description: Targeted sweep metrics now expose a histogram `batchSizeHistogram`
```

Comment:
I think this will mess up changelog, although could be wrong. I believe you want:

```
description: |
  Targeted sweep metrics...
```

IIRC the `|` is necessary for multiline strings.